cmd/eksctl-anywhere/cmd/generateclusterconfig.go

```diff
@@ -170,8 +170,14 @@ func generateClusterConfig(clusterName string) error {
 			}
 			datacenterYaml = dcyaml
 
-			cpMachineConfig := v1alpha1.NewTinkerbellMachineConfigGenerate(clusterName + "-cp")
-			workerMachineConfig := v1alpha1.NewTinkerbellMachineConfigGenerate(clusterName)
+			cpMachineConfig, err := v1alpha1.NewTinkerbellMachineConfigGenerate(clusterName + "-cp")
+			if err != nil {
+				return fmt.Errorf("error outputting yaml: %v", err)
```

Comment:
There's no need to add 'error' to error messages. As errors are wrapped with increasing amounts of context you end up with lots of 'error' and we fundamentally know its an error because of how it gets presented to the end user (e.g. `error exuting cmd: error with hardware yaml: error writing file: error file already exists`)

Reply:
I'll clean all of these up